x-pack/test/api_integration/apis/management/cross_cluster_replication/follower_indices.js

```diff
@@ -90,11 +90,7 @@ export default function ({ getService }) {
         const payload = getFollowerIndexPayload(leaderIndex);
         const { body } = await createFollowerIndexRequest(undefined, payload).expect(200);
 
-        expect(body).to.eql({
-          follow_index_created: true,
-          follow_index_shards_acked: true,
-          index_following_started: true
-        });
+        expect(body.follow_index_created).to.eql(true);
```

Comment:
Can we add a comment to explain why we're not checking any other properties, so someone doesn't accidentally reintroduce the flakiness?

```js
// There is a race condition in which Elasticsearch can respond without acknowledging,
// i.e. `body .follow_index_shards_acked` is sometimes true and sometimes false.
// By only asserting that `follow_index_created` is true, we eliminate this flakiness.
```